bin/setup_dot1x

```diff
@@ -27,16 +27,18 @@ echo -e "set IPs for faux"
 
 docker run -d --privileged --name daq-fr --hostname daq-fr daq/networking
 docker exec -ti daq-fr ip addr flush eth0
-docker exec -ti daq-fr ifconfig eth0 172.17.0.12/16
+ip=$(ip -o -4 addr list docker0 | awk '{print $4}' | cut -d '/' -f1)2
+docker exec -ti daq-fr ifconfig eth0 $ip
```

Comment:
don't need -ti in scripts

Reply:
Trevor. I think the fiel is outdated. The recent one doesnt have -ti